packages/plugin-core/assets/dendron-ws/templates/dendron.templates.daily.md

```diff
@@ -0,0 +1,43 @@
+_Edit the [[dendron.templates.daily]] note to change this template generated for Dendron Daily Journal._
+
+<!--
+Based on the journaling method created by Intelligent Change:
```

Comment:
Lets link it back to our own blog. Replace with 

> Based on the 5 minute journal method. See more details [here](https://blog.dendron.so/notes/P1DL2uXHpKUCa7hLiFbFA/)

Reply:
I feel linking to [[dendron.templates.daily]] is useful as it doesn't require them to read the tutorial to make any modifications. I added your line to the blog right underneath for those that are curious